tests/test_private_attributes.py

```diff
@@ -79,8 +79,6 @@ class Model(BaseModel):
 
         _foo: str
 
-        model_config = ConfigDict(underscore_attrs_are_private=True)
```

Comment:
This causes a user warning now, so I removed it. However, currently, the message just says it was removed, not providing the extra detail that "v2 behaves as though it was set to True everywhere". If we want to provide this level of detail I could modify the `ConfigDict.__init__` to have more specific error messages for each individual field.